### DISCUSSION TITLE
Creating a Team for an newly created SharePoint Site: Error: Insufficient privileges to complete the operation.

### DISCUSSION THREAD

**Discussion by myatix** [pnp/powershell#2643] [Q&A]
Hi All,

I am trying to create a Team for an existing site as part of a LogicApp/ Azure Function App.

I keep getting the following error.  Error: Insufficient privileges to complete the operation.

Function Code:

Try {
 
    #Temify the Group connected site
    $Site = Get-PnPTenantSite -Identity $webUrl
    New-PnPTeamsTeam -GroupId $Site.GroupId.Guid
}
Catch {
    write-host -f Red "Error:" $_.Exception.Message
}

API Permissions:

![image](https://user-images.githubusercontent.com/48433526/207329701-fd70db5e-4004-422b-bd88-018075a6a431.png)

**Comment by myatix** [comment]
Can anyone tell me if this is a known issue or am I doing something wrong... The rest of the FunctionApp works fine it is only when I get to the New-PnPTeamsTeam that I receive an Insufficient privilege's error?

**Comment by scaummb** [comment]
这是来自QQ邮箱的假期自动回复邮件。您好，我最近正在休假中，无法亲自回复您的邮件。我将在假期结束后，尽快给您回复。
